tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -185,22 +221,14 @@ describe(__filename, () => {
     const addon = addonForGrouping(grouping);
     store.dispatch(setGroupedRatings({ addonId: addon.id, grouping }));
     const root = render({ addon });
-    const counts = root.find('.RatingsByStar-count').find(Link);
-
-    function validateLink(link, score, expectedTitle) {
-      expect(link.children()).toHaveText(grouping[score].toString());
-      expect(link).toHaveProp(
-        'to',
-        reviewListURL({ addonSlug: addon.slug, score }),
-      );
-      expect(link).toHaveProp('title', expectedTitle);
-    }
-
-    validateLink(counts.at(0), '5', 'Read all five-star reviews');
-    validateLink(counts.at(1), '4', 'Read all four-star reviews');
-    validateLink(counts.at(2), '3', 'Read all three-star reviews');
-    validateLink(counts.at(3), '2', 'Read all two-star reviews');
-    validateLink(counts.at(4), '1', 'Read all one-star reviews');
+    const tableRow = root.find('.RatingByStar-table-row');
+    const counts = tableRow.find('.RatingsByStar-count');
+
+    expect(counts.at(0).children()).toHaveText(grouping['5'].toString());
```

Comment:
I wonder if you can just add these assertions into the test you changed above, as opposed to having a separate test for it. It seems like it's just testing one more thing that appears on a row. The existing test is also checking the links, but we no longer have links on each of these.

Reply:
Yes, we can do that. I will update that.